src/app/form/(formSteps)/personal-information/page.tsx

```diff
@@ -1,11 +1,460 @@
+                    {orderedInputNameToLabel["lastName"]}
+                  </Label>
+                  <TextInput
+                    id="lastName"
+                    data-testid="lastName"
+                    type="text"
+                    autoComplete="family-name"
+                    validationStatus={errors.lastName ? "error" : undefined}
+                    aria-invalid={errors.lastName ? "true" : "false"}
+                    aria-describedby={errors.lastName && "lastNameErrorMessage"}
+                    {...register("lastName", {
+                      required: `${orderedInputNameToLabel["lastName"]} is required`,
+                    })}
+                  />
+                  {errors.lastName && (
+                    <span id="lastNameErrorMessage" className="usa-error-message">
+                      {errors.lastName.message}
+                    </span>
+                  )}
+                </div>
+              </Fieldset>
+
+              <Fieldset legend="Date of birth" className="margin-top-3" requiredMarker>
+                <span className="usa-hint">For example: April 28 1986</span>
+                <DateInputGroup>
+                  <FormGroup className="usa-form-group--month usa-form-group--select">
+                    <Label htmlFor="dateOfBirthMonth" requiredMarker>
+                      {orderedInputNameToLabel["dateOfBirthMonth"]}
+                    </Label>
+                    <Select
+                      id="dateOfBirthMonth"
+                      required
+                      autoComplete="bday-month"
+                      validationStatus={errors.dateOfBirthMonth ? "error" : undefined}
+                      aria-invalid={errors.dateOfBirthMonth ? "true" : "false"}
+                      aria-describedby={errors.dateOfBirthMonth && "dateOfBirthMonthErrorMessage"}
+                      {...register("dateOfBirthMonth", {
+                        required: `${orderedInputNameToLabel["dateOfBirthMonth"]} is required`,
+                      })}
+                    >
+                      <option value="1">01 - January</option>
+                      <option value="2">02 - February</option>
+                      <option value="3">03 - March</option>
+                      <option value="4">04 - April</option>
+                      <option value="5">05 - May</option>
+                      <option value="6">06 - June</option>
+                      <option value="7">07 - July</option>
+                      <option value="8">08 - August</option>
+                      <option value="9">09 - September</option>
+                      <option value="10">10 - October</option>
+                      <option value="11">11 - November</option>
+                      <option value="12">12 - December</option>
+                    </Select>
+                  </FormGroup>
+                  <FormGroup className="usa-form-group--day">
+                    <Label htmlFor={"dateOfBrthDay"} requiredMarker>
```

Comment:
Typo has broken the `htmlFor` link to the associated input
```suggestion
                    <Label htmlFor={"dateOfBirthDay"} requiredMarker>
```